tmail-backend/integration-tests/jmap/postgres-jmap-integration-tests/src/test/java/com/linagora/tmail/james/PostgresEmailGetMethodTest.java

```diff
@@ -0,0 +1,22 @@
+package com.linagora.tmail.james;
+
+import static com.linagora.tmail.james.TmailJmapBase.JAMES_SERVER_EXTENSION_FUNCTION;
+import static com.linagora.tmail.james.TmailJmapBase.MESSAGE_ID_FACTORY;
+
+import org.apache.james.JamesServerExtension;
+import org.apache.james.jmap.rfc8621.contract.EmailGetMethodContract;
+import org.apache.james.jmap.rfc8621.contract.probe.DelegationProbeModule;
+import org.apache.james.mailbox.model.MessageId;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+public class PostgresEmailGetMethodTest implements EmailGetMethodContract {
```

Comment:
Just curious why we need this while we already have one in james?

Reply:
Because tmail has jmap extension,
 It overrides some modules,